source/_posts/2016-06-13-using-async-and-await-with-ember-cli-page-object.markdown

```diff
@@ -111,10 +111,16 @@ Take a look at [this commit](https://github.com/san650/tajpado/commit/6638a26564
 
 ## A note about JSHint
 
-JSHint doesn't like the new _async_ and _await_ keywords so it returns a handfull of errors. To disable JSHint on a particular file you can add a comment directive on top of the file.
+JSHint doesn't like the new keywords _async_ and _await_ so it returns a handfull of errors. To disable JSHint on a particular file you can add a comment directive on top of the file.
```

Comment:
And in the whole change I think keywords goes at the end. At least it should be consistent with https://github.com/wyeworks/blog/pull/35/commits/d637cb9eb4fdbc73d2eecef609bd7266f003ef21#diff-881e145733de402e9a68e44394539addR20 :)